.github/workflows/master.yml

```diff
@@ -12,30 +12,38 @@ jobs:
     runs-on: ubuntu-latest
     environment:
       name: octoboard-server
-      url: https://octo-board.appspot.com/
+      url: https://api.octoboards.com/
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - uses: ikuanyshbekov/app-yaml-env-compiler@v1.1
+      - name: Set up SSH
+        run: |
+          mkdir -p ~/.ssh/
+          echo "$SSH_PRIVATE_KEY" > ./deploy.key
+          sudo chmod 600 ./deploy.key
+          echo "$SSH_KNOWN_HOSTS" > ~/.ssh/known_hosts
+        shell: bash
         env:
-          SENTRY_DSN: ${{ secrets.APP_SENTRY_DSN }}
-          ENVIRONMENT: production
-      - name: Setup Cloud SDK
-        uses: google-github-actions/setup-gcloud@v0.2.1
-        with:
-          project_id: octo-board
-          service_account_key: ${{ secrets.APP_ENGINE_SERVICE_ACCOUNT_OCTO_BOARD }}
-      - name: Deploy to App Engine
-        run: gcloud app deploy --quiet --version=${GITHUB_SHA}
+          SSH_PRIVATE_KEY: ${{secrets.SSH_PRIVATE_KEY}}
+          SSH_KNOWN_HOSTS: ${{secrets.SSH_KNOWN_HOSTS}}
+      - name: Create env file
+        run: |
+          touch .env
+          echo PRODUCTION_HOST=${{ secrets.PRODUCTION_HOST }} >> .env
```

Comment:
Here, we're using secrets directly in run, but in a previous step we first assign them to env variables 👀

Reply:
Yup that may be the case. I'd use env variables everywhere for consistency